Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -62,7 +62,12 @@ moduleConfigProperty :
 ;
 
 moduleConfigElement :
-    unrestrictedIdentifier whiteSpace? EQ whiteSpace? expression (COLON numberLiteral)? endOfStatement
+    lExpression whiteSpace? EQ whiteSpace? (shortcut | DOLLAR? expression) (COLON (numberLiteral | IDENTIFIER))? endOfStatement
```

Comment:
Did you test this on a case where the hex value starts with a number and contains a letter?